CDCgov/prime-reportstream
# Cle/6038 send to stlt

This PR ensures that a universal pipeline messages makes it all the way through to the 'send' step with the correct task, action, and report_file records.

Test Steps:
1. Run unit tests
OR
1. Send a universal pipeline message
2. Check DB to see that there are steps for receive, convert, route, translate, batch, send

## Changes
- add new 'convert' TaskAction
- update next step on receive to be 'convert' when it is universal pipeline
- update batch function to work for both covid and UP
- update batch function to make it testable with injectable mocked components
- nullify next_action on task after it happens for UP, to match covid pipeline activities
- insert tasks and actions for each step
- create hl7MessageRSHelpers to take an HL7 message, create a RS Report based off of it, and add proper lineage
- add batch function tests for UP. Note: There were not tests for covid batch and still are not. Tech debt.

## Checklist

### Testing
